antidotum/tergo/src/rust/src/lib.rs

```diff
@@ -72,7 +72,15 @@ fn format_code(source_code: &str, configuration: extendr_api::List) -> String {
             .unwrap_or(default_config.insert_newline_in_quote_call.0),
     );
 
-    tergo_lib::tergo_format(source_code, Some(&config)).unwrap()
+
+    match tergo_lib::tergo_format(source_code, Some(&config)) {
```

Comment:
BTW, the issue of panics stemming from wrong configuration values still persists. It's all lines 15-75.